app/services/page_creator.rb

```diff
@@ -13,23 +13,11 @@ def initialize(params)
   def run
     @page = Page.find @page_id
     Donation.run(@page, @params)
-    create_donation_form
+    Petition.run(@page, @params)
     PageFollowUpCreator.run(
-      page_ak_uri:   @page.ak_donation_resource_uri,
-      language_code: @page.language.try(:code)
+        page_ak_uri:   @page.ak_donation_resource_uri,
+        language_code: @page.language.try(:code)
```

Comment:
is this indentation normal?

Reply:
Ah, it is not! I'll go through and fix where I have four space indentation.